packages/claims/src/claims/claims.ts

```diff
@@ -38,57 +47,89 @@ export class Claims implements IClaims {
   }
 
   /**
-   * Verifies signers signature on received token
-   * @example
-   * ```typescript
-   * import { Keys } from '@ew-did-registry/keys';
-   * import { Claims } from '@ew-did-registry/claims';
-   *
-   * const user = new Keys();
-   * const claims = new Claims(user);
-   * const verified = claims.verifySignature(token, userDid);
-   * ```
-   *
-   * @param { string } token token signature on which you want to check
-   * @param { string } signer did of the signer
-   */
-  async verifySignature(token: string, signer: string): Promise<boolean> {
-    const signerPubKey = await this.document.ownerPubKey(signer);
-    try {
-      await this.jwt.verify(token, signerPubKey as string);
-    } catch (error) {
-      return false;
-    }
-    return true;
-  }
-
-  /**
-   * Verifies integrity of the claim, the claim is issued by the user
-   *  delegate and the authenticity of the issuer's signature
+   * Verifies issuance and publishing of claim at `claimUrl`.
+   * On success returns claim.
+   * Verification takes into account purpose on which claim was issued.
+   * Claim can be issued to authenticate identity or to assert issuer approval.
+   * By default asserts claim approval. 
    *
    * @param claimUrl {string}
    * @param hashFns {{ [alg: string]: (data: string) => string }}
```

Comment:
Could you update the documentation? I think we should add the new properties and I don't think we need the type in the docs 🤔  
```suggestion
   * @param claimUrl: Url of claim to be verified. This method will retrieve the claim using the didStore configured for the class
   * @param params.hashFns: The function used to determine the of hash of the claim token used in the DID document. Used to verify that the DID document service endpoint matches the retrieved claim.
   .... other params
```